libkbfs/folder_block_manager.go

```diff
@@ -1101,7 +1110,10 @@ func (fbm *folderBlockManager) doReclamation(timer *time.Timer) (err error) {
 	}
 	if len(ptrs) == 0 && !shortened {
 		complete = true
-		return nil
+
+		// Add a new gcOp to show other clients that they don't need
+		// to explore this range again.
+		return fbm.finalizeReclamation(ctx, nil, nil, latestRev)
```

Comment:
I'm not super happy about this since it clutters up the MD history a bit. But we can fix that later anyway, right?

Let me see if I understand the scenario here. Say a user puts revisions 1-100. QR kicks in for all but the last one, and puts revisions 101 (QRing 1-25), 102 (QRing 26-50), 103 (QRing 51-75), and 104 (QRing, 75-100). Then if the user doesn't do anything else, clients will have to fetch revisions 101-104 to conclude that QR is caught up. Then this PR makes the first client to do so rev. 105 ("QRing" 101-104).

Ideally, though, we'd have 104 know to mark 75-104, right? I guess that might turn out to be complicated in edge cases where two clients are QRing at the same time.

Do I understand it correctly?